src/category_theory/abelian/transfer.lean

```diff
@@ -189,4 +198,157 @@ def abelian_of_equivalence
   (F : C ⥤ D) [functor.preserves_zero_morphisms F] [is_equivalence F] : abelian C :=
 abelian_of_adjunction F F.inv F.as_equivalence.unit_iso.symm F.as_equivalence.symm.to_adjunction
 
+section transfer_enough_injectives
+
+/-!
+If `L ⊣ R` are a pair of adjoint functors between abelian categories `𝓐` and `𝓐` and `L` is
+faithful and exact, then if `𝓑` has enough injectives, so does `𝓐`. We achieve this by considering
+an arbitrary injective presentation of `L(A) ⟶ J`: by adjunction, there is an `A ⟶ R(J)`, we will
+prove that this `A ⟶ R(J)` is an injective presentation of `A`.
+-/
+
+open limits adjunction
+
+universes v₁ v₂
+
+variables {C : Type u₁} {D : Type u₂} [category.{v₁} C] [category.{v₂} D] [enough_injectives D]
+variables (L : C ⥤ D) (R : D ⥤ C)
+
+namespace enough_injectives_of_adjunction_auxs
+
+/--
+Given an injective presentation `L(A) → J`, then `injective_object_of_adjunction A` is defined to be
+`R(J)`. It will later be proven to be an injective object in `𝓐`.-/
+def RJ (A : C) : C := R.obj $ injective.under (L.obj A)
```

Comment:
This docstring seems to mention a nice and readable name for `RJ`. On the other hand, that name suggests that the defn needs an adjunction, which it doesn't.
Please look at the names in this PR, and see if you can optimize them a bit.